tools/ssl-create.sh

```diff
@@ -85,12 +85,21 @@ if [ ! -r "${PEM_FINAL_CERT}" ] || [ ! -s "${PEM_FINAL_CERT}" ] || \
         exit $?
     fi
 
+    # certificate configuration (disable CA)
+    echo "[ req ]" > /run/tntnet-bios/ssl.conf
+    echo "distinguished_name=dn" >> /run/tntnet-bios/ssl.conf
+    echo "[ dn ]" >> /run/tntnet-bios/ssl.conf
+    echo "[ ext ]" >> /run/tntnet-bios/ssl.conf
+    echo "basicConstraints=CA:FALSE" >> /run/tntnet-bios/ssl.conf
+
     # Generate self-signed cert with a new key and other data
-    openssl req -x509 -sha256 -newkey rsa:2048 \
+    openssl req -config /run/tntnet-bios/ssl.conf -x509 -sha256 -newkey rsa:2048 \
         -keyout "${PEM_KEY}" -out "${PEM_CERT}" \
-        -days 365 -nodes -subj "/CN=${LOCAL_HOSTNAME}" \
+        -days 9125 -nodes -subj "/CN=${LOCAL_HOSTNAME}" \
     || exit $?
 
+    rm /run/tntnet-bios/ssl.conf
```

Comment:
This `rm` does not happen if we exit above with openssl error. Do we want to keep the config for postmortems then? Or should we `rm` along with the exit: `... || { RES=$?; rm -f ... ; exit $RES; }` and/or `trap` an exit from the script?